### ISSUE TITLE
Support additional OSX icons

### ISSUE THREAD

**Issue by mattrayner** [nickcoutsos/keymap-editor#220]
Hi, it would be cool if we could show OSX icons for additional keys:

- Brightness Up (pause break)
- Brightness Down (scroll lock)

Im happy to contribute a PR for this but have seen the closed source change you are making...

**Comment by nickcoutsos** [comment]
Good call, I should include the brightness keys. I'm using FontAwesome currently and taking a quick peek at the collection it seems that brightness up/down aren't included in the free set. I've had other such limitations and have been meaning to explore other icon sets so this is just more justification for that.

Could you clarify what you mean with the pause break and scroll lock in your parentheses? I'm looking at the [ZMK Keycodes: Display](https://zmk.dev/docs/codes#display) table which just references HID codes. I'm on my MacBook where brightness are dedicated keys (touchbar, actually, bleh), are there fullsize external keyboards where those functions share the same keys?

**Comment by mattrayner** [comment]
For OSX you typically use the keys SLCK for brightness down and PUSE_BREAK for brightness up as those keys are mapped by default at the OS level to brightness up and down

**Comment by mattrayner** [comment]
Personally I'm moving to keymap-editor from Oryx and am just noting the 'differences', originally with the plan to commit back and improve, still happy to do that onto this repo if you wanted to then pull those changes forward onto your release?

**Comment by mattrayner** [comment]
For example, C_NEXT and C_PREV would be good to have an icon for:

<img width="330" alt="image" src="https://github.com/nickcoutsos/keymap-editor/assets/3502902/f2e7a050-383a-4433-b646-be732dae6b4d">

**Comment by mattrayner** [comment]
Backlight controls could too:

<img width="398" alt="image" src="https://github.com/nickcoutsos/keymap-editor/assets/3502902/78806ff4-ea57-4f09-a433-051e51a13ab0">

Along with the Bluetooth control icons

Have you looked at material design icons (MDI) [Material Design Icons](https://pictogrammers.com/library/mdi/)
